src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  let partnersText = 'I am not married';
+
+  if (isMarried === true && sex === 'm') {
+    partnersText = `${partnerName} is my wife`;
+  } else if (isMarried === true && sex === 'f') {
+    partnersText = `${partnerName} is my husband`;
+  } else {
+    partnersText = 'I am not married';
+  }
```

Comment:
- Try to simplify it and get rid of `if`/`else if`/`else` statements
- You don't need to compare with boolean variables as `isMarried` is already boolean